.github/workflows/run-tests.yml

```
@@ -0,0 +1,85 @@
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        api-level: [30] # Android API level for the emulator
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Set up JDK 17
+        uses: actions/setup-java@v3
```

actionlint (1.7.4)

the runner of "actions/setup-java@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)